docs/micro-lc/overview.md

```diff
@@ -111,8 +115,8 @@ The configuration consumed by the **fe-container** is served by **be-config**, d
 ## Plugins
 
 The **plugins** are the `micro-fronted` that actually composes the end application.  
-Each plugin is an **independent**, **self-contained** entity which can be written **in any framework**:
-plugins can have their own dedicated back-end, and they can be extended or configured at will at runtime.
+Each plugin is an **independent**, **self-contained** entity which can be written **in any framework**.
+Plugins can have their own dedicated backend, and they can be extended or configured at will at runtime.
```

Comment:
same as front-end

Reply:
I tried to be consistent and adapt the word `backend` to `frontend` (see explanation above)